### ISSUE TITLE
Pressing double space to open full screen vis is closing it first

### ISSUE THREAD

**Issue by sylwiabr** [enso-org/enso#6260]
### What type of issue is this?

Transient – Occuring only once

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

Having the visualization under the node open and pressing double space to have it full screen is closing it and then reopening. This causes the settings of the visualization (like custom columns width for table) to reset.

### Expected behaviour

Opening the full screen visualization should not close it.

### Enso Version

2023.1.1-nightly.2023.4.13

### Browser or standalone distribution

Standalone distribution

### Browser Version or standalone distribution

standalone

### Operating System

MacOS

**Comment by sylwiabr** [comment]
This is rather old regression so there is no point in bisecting.

**Comment by farmaazon** [comment]
Is it actually a regression? @sylwiabr Do you remember it working at any point? Because I think it always worked this way.

We may check even the last alpha package.

**Comment by sylwiabr** [comment]
@wdanilo mentioned we were handling it specially to preserve such behavior.

**Comment by farmaazon** [comment]
The double pressing is handled in the general EnsoGL keyboard handling system. I haven't found any code suppressing single press when double press is expected.

Also, I don't know how to handle it properly. We usually do not want to delay GUI reactions for keystrokes - and this would be necessary if we want to suppress single press event when double pressed event is fired. So probably we need to handle this in visualization container. But here too, I think, delaying hiding visualization on single space press would give a bad UX. Perhaps we could restrain from deleting visualization instance for some time after hiding visualization (so the state like changed column width will be preserved). 

However, maybe we should somehow preserve visualization state not only in "switching to full screen" scenario?

**Comment by wdanilo** [comment]
100% agree with Adam, this would be a bad UX. Hmm, @farmaazon @sylwiabr do you have proposals of how it might work? Let's consider ANY change, including changing shortcuts.

**Comment by sylwiabr** [comment]
I would suggest `shift+space` for full screen vis - what do you. think @wdanilo? Also an icon to open/close full screen vis would be a grat improvement but there is a separate task for this.

**Comment by wdanilo** [comment]
Let's do it :)
